module/zfs/dsl_scan.c

```diff
@@ -68,18 +68,6 @@
  * queue and issue them out of logical order and in a way that most benefits
  * physical disks (LBA-order).
  *
- * This sorting algorithm is subject to limitations. We can't do this with
- * blocks that are non-leaf, because the scanner itself depends on these
- * being available ASAP for further metadata traversal. So we exclude any
- * block that is bp_level > 0. Fortunately, this usually represents only
- * around 1% of our data volume, so no great loss.
- *
```

Comment:
Thanks for the clarification.  With minimal belaboring of the point, is it fair to say that the scan phase is using ordinary reads, and so benefits from read parallelism on mirrors, while the scrub phase, well, scrubs, and so will be rate-limited by the slowest drive in a mirror?

Thanks again for all your work on this. :)